engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionBuilder.scala

```diff
@@ -393,7 +409,8 @@ final class SuggestionBuilder[A: IndexedSource](
       args               = Seq(thisArg),
       doc                = None,
       typeSignature      = argument.name.getMetadata(TypeSignatures),
-      genericAnnotations = None
+      genericAnnotations = None,
+      methodType         = MethodType.Getter
```

Comment:
There already is `buildGetter` method and we just make sure it constructs `Getter` and not just `Method`, rigtht?

Reply:
Yes, here we specify that `builtGetter` makes a getter